packages/jest-types/src/Config.ts

```diff
@@ -313,7 +333,8 @@ export type ProjectConfig = {
   dependencyExtractor?: string;
   detectLeaks: boolean;
   detectOpenHandles: boolean;
-  displayName: string | null | undefined;
+  displayName?: string;
```

Comment:
I wonder if we should do `displayName?: string | {name: string; color: DisplayNameColor}` instead of adding a new option?

Reply:
Oh, I like this implementation better actually.

Reply:
I like the API that @SimenB is suggesting.